subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/src/fs_mgmt.c

```diff
@@ -54,33 +59,31 @@ fs_mgmt_file_download(struct mgmt_ctxt *ctxt)
 {
 	uint8_t file_data[FS_MGMT_DL_CHUNK_SIZE];
 	char path[CONFIG_FS_MGMT_PATH_SIZE + 1];
-	unsigned long long off;
-	CborError err;
+	unsigned long long off = ULLONG_MAX;
```

Comment:
can uint64_t be used now that tinycbor has done?

Reply:
Fixed.